Failure on aarch64-linux [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34532&attempt_id=a2bcf790-ffd0-4500-b2eb-72bd20d363e1)

<details><summary>Partial log (click to expand)</summary><p>

```
Usage: test-regidx [OPTIONS]
Options:
   -h, --help          this help message
   -s, --seed <int>    random seed
   -v, --verbose       increase verbosity by giving multiple times
make: *** [Makefile:247: test-plugins] Error 1
builder for '/nix/store/2k4xwjhxa1650x0x5gs8pf4ixhkkqzpx-bcftools-1.6.drv' failed with exit code 2
cannot build derivation '/nix/store/xdkizrhkgxqjjawd8agwnq8wy38yjm0d-python2.7-pysam-0.13.0.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/7ig76c3587bhs2sx9y6qdvkzp27d1spk-python3.6-pysam-0.13.0.drv': 1 dependencies couldn't be built
error: build of '/nix/store/7ig76c3587bhs2sx9y6qdvkzp27d1spk-python3.6-pysam-0.13.0.drv', '/nix/store/xdkizrhkgxqjjawd8agwnq8wy38yjm0d-python2.7-pysam-0.13.0.drv' failed
```
</p></details>

--- a/pkgs/development/python-modules/pysam/default.nix
+++ b/pkgs/development/python-modules/pysam/default.nix
@@ -1,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, bzip2
+, bcftools
+, curl
+, cython
+, htslib
+, lzma
+, pytest
+, samtools
+, zlib
+}:
+
+buildPythonPackage rec {
+  pname   = "pysam";
+  version = "0.13.0";
+
+  # Fetching from GitHub instead of PyPi cause the 0.13 src release on PyPi is
+  # missing some files which cause test failures.
+  # Tracked at: https://github.com/pysam-developers/pysam/issues/616
+  src = fetchFromGitHub {
+    owner = "pysam-developers";
+    repo = "pysam";
+    rev = "v${version}";
+    sha256 = "1lwbcl38w1x0gciw5psjp87msmv9zzkgiqikg9b83dqaw2y5az1i";
+  };
+
+  buildInputs = [ bzip2 curl cython lzma zlib ];
+
+  checkInputs = [ pytest bcftools htslib samtools ];
+
+  checkPhase = "py.test";
+
+  preInstall = ''
+    export HOME=$(mktemp -d)
+    make -C tests/pysam_data
+    make -C tests/cbcf_data
+  '';
+
+  meta = {
+    homepage = http://pysam.readthedocs.io/;
+    description = "A python module for reading, manipulating and writing genome data sets";
+    maintainers = with lib.maintainers; [ unode ];
+    license = lib.licenses.mit;
+    platforms = [ "i686-linux" "x86_64-linux" ];
+  };
+}

--- a/pkgs/top-level/python-packages.nix
+++ b/pkgs/top-level/python-packages.nix
@@ -13226,6 +13226,8 @@ in {
     };
   };
 
+  pysam = callPackage ../development/python-modules/pysam { };
+
   pysaml2 = buildPythonPackage rec {
     name = "pysaml2-${version}";
     version = "3.0.2";

